src/components/ZetkinCalendar/MonthCalendar/index.tsx

```diff
@@ -125,6 +136,7 @@ const MonthCalendar = ({ orgId, campaigns, baseHref, events, focusDate, tasks }:
                         ...daysEvents.map(e => ({ data: e, id: 'event' })),
                     ];
                     const totalTasksAndEvents = tasksAndEvents.length;
+                    const isToday = isInRange(today, currentDate, tomorrow);
```

Comment:
This might be overkill, maybe @richardolsson has an opinion, but, 

It's wasteful to call the isInRange function (30-odd times) when we know today isn't in the calendar view.
Maybe add logic in the top of the component that checks if today is in View and only call this function then?

smth like
`todayInView = firstCalendarDay > today > lastCalendarDay`

Reply:
It's a good suggestion, but let's leave it as it is for now. 😊